### DISCUSSION TITLE
Loki Datasource: Dashboard variable of type: Query has no Query-type: Query Result

### DISCUSSION THREAD

**Discussion by samjewell** [grafana/grafana#68516] [Data Sources]
**What happened**:

I'm building a dashboard where I need to use the results of a Logs query as the input to a Metrics query.

Specifically there are Log-lines which identify which users are hitting a sad-path in the application. I need to join these results together with a dataset present in a metrics datasource. Unfortunately the metrics datasource is so large that I can only get results back by passing the user-ids as part of the query itself. Hence I need to know the user-ids before trying to render the panel, so I need to gather those IDs in a dashboard-variable (because these are evaluated first)

Prometheus datasource offers this option:

![image](https://github.com/grafana/grafana/assets/2903904/4721f7af-63c0-4450-84e9-1efdee838544)

But Loki datasource doesn't:

![image](https://github.com/grafana/grafana/assets/2903904/9d397d61-28ea-4aea-af37-24cdf62e7676)

And the data that I need isn't present in either "label names" or "label values"

**What did you expect to happen**:

Query-type: Query-result to be present in the dashboard-variables config for Loki

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82693. Feel free to continue the discussion around this there. Thank you!

**Reply by samjewell** [reply]
@gelicia I'm curious - how was it that this one was chosen to be migrated to an issue by the team, but many of my other discussions (that had more comments and upvotes) were not? It might help me to respond better, if I can understand how the team are making decisions and why.

**Reply by gelicia** [reply]
@samjewell I sent you the internal documents but just for anyone else wondering.. the criteria was to label discussions as old (prior to jan 2023) or new, then go by the number of engagements. That said, there was a bug on some early runs of my [discussion list export](https://github.com/gelicia/discussion-exportinator)  where it didn't count upvotes. Since all those would have ran into the same "closed" bucket that keeps it from showing up in my query, it would have been non-trivial to isolate them. I decided to move forward with the migration as is.

If any of your discussions were closed and you would like me to migrate them, I can. Just leave a comment and I will get notified. Thank you!
